README.md

```diff
@@ -1,18 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> One paragraph statement about the project.
+> First Module project to master the tools necessary to complete industry standard development projects using best-practices and tools.
 
 ![screenshot](./app_screenshot.png)
```

Comment:
- [ ] To make `README` look professional, please consider deleting `screenshot` since there is no `app_screenshot.png` file in the project.